core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/requests/ImageCatalogV4Request.java

```diff
@@ -4,11 +4,25 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.imagecatalog.base.ImageCatalogV4Base;
+import com.sequenceiq.cloudbreak.doc.ModelDescriptions;
 
 import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
 
 @ApiModel
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(Include.NON_NULL)
 public class ImageCatalogV4Request extends ImageCatalogV4Base {
+
+    @ApiModelProperty(ModelDescriptions.CRN)
+    private String resourceCrn;
```

Comment:
why we have crn on the request?

Reply:
because there is a update endpoint which needs a crn as a unique identifier